lib/documents_tagger.rb

```diff
@@ -32,10 +32,22 @@ def mapped_taxons(document_type)
     document_type.camelize.constantize.new.taxons
   end
 
+  def get_paged_editions_enum(params = {})
+    enum = Services.publishing_api.get_paged_editions(params)
+    Enumerator.new do |yielder|
+      enum.each do |response|
```

Comment:
`enum` isn't very clear to me. I guess it is a page? If so, `pages` might read better as a variable name?